### DISCUSSION TITLE
ESPN + guide data

### DISCUSSION THREAD

**Discussion by goldengate2121** [K-vanc/Tempest-EPG-Generator#176] [Ideas]
Is it possible to build a siteconfig and channel file for ESPN+. Looks like IPTV Boss has it but I don't know where they pull the data from.

**Comment by K-vanc** [comment]
Hi, for requests, you need to provide me an url of working epg
website/source of this channel otherwise it is being a wish instead of
request. I made a Quick check on Google but there is nothing can be used as
a epg source. Maybe they are pulling from some main epg providers such as
schedulesdirect or other usa network providers. If you dont know a source,
you may try ur luck by checking US or Multi Nation folder siteconfigs. Some
of them creating channel lists with zip codes and may have that channel.

and please use Issue section for this kind of questions instead of discussions
